sdk/extensions/Microsoft.Extensions.Azure/src/Internal/ClientFactory.cs

```diff
@@ -158,6 +161,14 @@ internal static TokenCredential CreateCredential(IConfiguration configuration)
                     workloadIdentityOptions.TokenFilePath = tokenFilePath;
                 }
 
+                if (additionallyAllowedTenantsList != null)
```

Comment:
Should we add test coverage for this change also?

Reply:
Fair point.  I forgot that I added this when passing through.